### PR TITLE
Add a crab to the code :crab: 

### PR DESCRIPTION
**Pull request by VanillaSylv**
Crabs in games make our game run better, as its a confirmed fact I've added a Crab to the code.

this aint serious but crab is funny

### CODE REVIEW

**Comment by alphaqu** [comment]
I have some concerns about the crab, The crab might be too good and freak windows out and may actually lead to worse performance as people now need to move at MACH 7 instead of 6 for taking their drink when mc is launching...

**Comment by pluiedev** [comment]
is fabric.mod.json parsed with a json5 parser? 🤔 the vanilla json spec doesn't allow comments, btw

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -41,3 +41,4 @@
     "hydrogen": "*"
   }
 }
+
```
Comment (Tacodiva) [inline src/main/resources/fabric.mod.json:44]: This white space shouldn't be added in this commit. It's important commits like this are high quality and don't contain unnecessary changes.

File: src/main/java/net/quantumfusion/dashloader/DashLoader.java [hunk 2]
```diff
@@ -448,7 +448,7 @@ private void initSerializers() {
         ThreadHelper.exec(runnables);
         LOGGER.info("[3/4] [" + Duration.between(start, Instant.now()).toMillis() + "ms] Created Serializers.");
     }
-
+// le crabe 🦀
```
Comment (leo60228) [inline src/main/java/net/quantumfusion/dashloader/DashLoader.java:451]: The whitespace should probably be changed to be a bit more consistent.
```suggestion
    
    // le crabe 🦀
    
```

File: src/main/resources/fabric.mod.json (hunk 1 above)
Comment (leo60228) [reply src/main/resources/fabric.mod.json:44]:
```suggestion
```

**Review by leo60228 — APPROVED** [review]
Looks good overall. I've included some whitespace changes that you should be able to accept via GitHub's web UI.

**Comment by alexmozaidze** [comment]
This pr is absolute gold

**Comment by VanillaSylv** [comment]
yea im closing this fucking pr because i keep getting emails about it

**Comment by Tacodiva** [comment]
You can click unsubscribe on the right you don't have to close
